jboss/container/wildfly/launch/security-domains/tests/login-modules.bats

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bats
+# dont enable these by default, bats on CI doesn't output anything if they are set
+#set -euo pipefail
+#IFS=$'\n\t'
+export BATS_TEST_SKIPPED=
+
+# fake JBOSS_HOME
+export JBOSS_HOME=$BATS_TMPDIR/jboss_home
+rm -rf $JBOSS_HOME 2>/dev/null
+mkdir -p $JBOSS_HOME/bin/launch
+
+# copy scripts we are going to use
+cp $BATS_TEST_DIRNAME/../../../launch-config/config/added/launch/openshift-common.sh $JBOSS_HOME/bin/launch
+cp $BATS_TEST_DIRNAME/../../../launch-config/os/added/launch/launch-common.sh $JBOSS_HOME/bin/launch
+cp $BATS_TEST_DIRNAME/../../../../../../test-common/logging.sh $JBOSS_HOME/bin/launch
+cp $BATS_TEST_DIRNAME/../added/launch/login-modules-common.sh $JBOSS_HOME/bin/launch
+mkdir -p $JBOSS_HOME/standalone/configuration
+
+# Set up the environment variables and load dependencies
+WILDFLY_SERVER_CONFIGURATION=standalone-openshift.xml
+
+# source the scripts needed
+source $JBOSS_HOME/bin/launch/openshift-common.sh
+source $JBOSS_HOME/bin/launch/login-modules-common.sh
+source $JBOSS_HOME/bin/launch/logging.sh
+
+setup() {
+  cp $BATS_TEST_DIRNAME/../../../../../../test-common/configuration/standalone-openshift.xml $JBOSS_HOME/standalone/configuration
+  sed -i "s|<!-- ##OTHER_LOGIN_MODULES## -->| |" "$CONFIG_FILE"
+}
+
+teardown() {
+  if [ -n "${CONFIG_FILE}" ] && [ -f "${CONFIG_FILE}" ]; then
+    rm "${CONFIG_FILE}"
+  fi
+}
+
+## test based on CLI operations
+normalize_spaces_new_lines() {
+    echo "output=${output}<<"
+  echo "expected=${expected}<<"
+  output=$(printf '%s\n' "$output" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//' -e '/^$/d')
+  expected=$(printf '%s\n' "$expected" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//' -e '/^$/d')
+  
+  #echo "output=${output}<<"
+  #echo "expected=${expected}<<"
+}
+
+@test "Configure CLI other security-domain login module " {
+  expected=$(cat <<EOF
+    if (outcome != success) of /subsystem=security:read-resource
+          echo "You have set environment variables to add a login module to other security domain. However, your base configuration doesn't contain the security subsystem. Fix your configuration for that to happen." >> \${error_file}
```

Comment:
Bats tests are failing, error messages differ from the error messages coded in the scripts: https://github.com/wildfly/wildfly-cekit-modules/pull/77/files#diff-ad09c5213a189e1b81b540e96fe91890R46

"You are adding a login module to other security domain" vs "You have set environment variables to add"